src/components/state-badge.component.js

```diff
@@ -13,28 +13,30 @@ type Props = {
   locale: string,
 };
 
-const styles = StyleSheet.create({
-  badge: {
-    padding: 12,
-    paddingTop: 3,
-    paddingBottom: 3,
-    borderRadius: 20,
-  },
-  mergedIssue: {
-    backgroundColor: colors.purple,
-  },
-  openIssue: {
-    backgroundColor: colors.green,
-  },
-  closedIssue: {
-    backgroundColor: colors.red,
-  },
-  text: {
-    fontSize: normalize(12),
-    ...fonts.fontPrimarySemiBold,
-    color: colors.white,
-  },
-});
+const colorOfIssueState = issueState => {
+  switch (issueState) {
+    case 'merged':
+      return colors.purple;
+    case 'open':
+      return colors.green;
+    case 'closed':
+      return colors.red;
+    default:
+      return colors.black;
+  }
+};
+
+const Badge = styled.View`
+  padding: 3px 12px;
+  border-radius: 20px;
+  background-color: ${({ issueState }) => colorOfIssueState(issueState)};
+`;
+
+const BadgeText = styled.Text`
+  font-size: ${normalize(12)};
+  font-family: ${fonts.fontPrimaryBold.fontFamily};
```

Comment:
Instead of:
`font-family: ${styledFonts.fontPrimaryBold};`

use:
`${fonts.fontPrimaryBold}`